doc/tutorial/erda.rst

```diff
@@ -131,23 +139,52 @@ MPI Oriented Development and Investigation (MODI)
 
 In order to install Veros with the `veros-bgc biogeochemistry plugin <https://veros-bgc.readthedocs.io/en/latest/>`__ start an **Ocean HPC Notebook** from the **Jupyter service** home page following :ref:`the instructions above <erda-jupyter>`.
 
-1. Change your current directory to ~/modi_mount by double-clicking the modi_mount folder circled in red
+1. Define in the **Terminal** where the package(s) with Veros dependencies should be installed. To do so, create the corresponding environment variable and the directory for the packages::
+
+      $ export CONDA_PKGS_DIRS=~/modi_mount/conda_dir
+      $ mkdir $CONDA_PKGS_DIRS
+
```

Comment:
Not necessary if we put the whole environment into `modi_mount` (and install only once)